src/ucp/proto/proto_am.inl

```diff
@@ -86,6 +127,14 @@ ucs_status_t ucp_do_am_bcopy_multi(uct_pending_req_t *self, uint8_t am_id_first,
             packed_len = uct_ep_am_bcopy(uct_ep, am_id_first, pack_first, req, 0);
             UCS_PROFILE_REQUEST_EVENT_CHECK_STATUS(req, "am_bcopy_first",
                                                    packed_len, packed_len);
+            if (am_flags != 0) {
+                if (ucs_unlikely(packed_len == UCS_ERR_NO_RESOURCE)) {
+                    if (ucs_unlikely(
+                        ucp_am_copy_header_to_internal_buf(req) != UCS_OK)) {
+                        return (ucs_status_t)UCS_ERR_NO_MEMORY;
```

Comment:
maybe return error status of `ucp_am_copy_header_to_internal_buf`?

Reply:
Yes. For this I will need a variable or use existing one.